rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -705,6 +718,11 @@ error_code cellGameCreateGameData(vm::ptr<CellGameSetInitParams> init, vm::ptr<c
 		return CELL_GAME_ERROR_NOTSUPPORTED;
 	}
 
+	if (prm->exists)
+	{
+		return CELL_GAME_ERROR_EXIST;
```

Comment:
Is this what a real system does?
It doesn't seem 'right'.  My expectation is that there would never exist game data here, since it is only a temporary directory, and hence would be unique for each invocation.
It is on the game itself to later perform the rename/move of the valid content to be within the content permit section of the disk.

Reply:
I will post hw tests later, the pr is not complete yet.